src/utils/cookie.ts

```diff
@@ -0,0 +1,110 @@
+import Cookies from 'js-cookie';
+import { SESSION_COOKIE_EXPIRATION_DURATION_MS } from '../constants/constants';
+
+export interface Session {
+  id: string;
+  token: string;
+  createdAt: number;
+}
+
+export const COOKIE_KEYS = {
```

Comment:
Would it make sense to use a const enum? If it doesn't need to be exported out of the repo (otherwise nevermind, we would hit the same compilation issues elsewhere)

Reply:
Yeah it's used in frontend interfaces... so let's wait a bit before transforming it into an enum.